torch_dvf/transforms.py

```diff
@@ -23,9 +24,11 @@ def __init__(
         rel_sampling_ratios: Tuple[float],
         interp_simplex: str,
         cluster_radii: Optional[Tuple[float]] = None,
+        max_num_neighbors: int = 32,
     ):
         self.rel_sampling_ratios = rel_sampling_ratios
         self.interp_simplex = interp_simplex
+        self.max_num_neighbors = max_num_neighbors
```

Comment:
American English, I see...

Reply:
don't like it tbh but `torch_geometric` uses such naming so better stick to the same one - and suffer american english xd